cogs/respects.py

```diff
@@ -15,6 +15,7 @@
           ":purple_heart:", ":blue_heart:"]
 DEFAULT_CH_DICT = {KEY_MSG : None, KEY_TIME : None, KEY_USERS : []}
 TIME_BETWEEN = timedelta(seconds=30) # Time between paid respects.
+MESSAGES = 20 # The number of messages in between

```

Comment:
20 seems a bit much, unless this was mostly for testing purposes.

Reply:
20 messages was roughly the number of single-line messages for the last respect to scroll out of view on my screen.  Definitely will put this in a config file in the next PR, as you suggested.

Reply:
Yea. Well, I feel like 20 messages would be a good amount of messages for a different "Press F to respect" to occur...